db/schema.rb

```diff
@@ -12,7 +12,7 @@
 
 ActiveRecord::Schema.define(version: 2021_10_23_185328) do
 
```

Comment:
@michellexliu I think you should un-checkin this commit, because you are currently using MySQL and it added more options